README.md

```diff
@@ -13,6 +13,11 @@ Prometheus.
 
 ## Non-goals
 
+First of all, the Pushgateway is not capable of turning Prometheus into a
+push-based monitoring system. For a general description of use cases for the
+Pushgateway, please read [When To Use The
+Pushgateway](https://prometheus.io/docs/practices/pushing/).
```

Comment:
in this cited doc, should https://github.com/RobustPerception/PushProx be mentioned in "Alternative strategies"?

Reply:
I like the idea. Would you like to open a PR against prometheus/docs?

Reply:
https://github.com/prometheus/docs/pull/1505